src/packages/core/extension-registry/models/file-upload-preview.model.ts

```diff
@@ -0,0 +1,11 @@
+import type { UmbFileUploadPreviewElement } from '../interfaces/file-upload-preview.interface.js';
+import type { ManifestElement } from '@umbraco-cms/backoffice/extension-api';
+
+export interface ManifestFileUploadPreview extends ManifestElement<UmbFileUploadPreviewElement> {
+	type: 'fileUploadPreview';
+	/**
+	 * Array of the mime types that are supported by this extension.
+	 * @example ["image/png", "image/jpeg", "image/*"]
+	 */
+	forMimeTypes: Array<string>;
```

Comment:
It looks like we have used the type `string | Array<string>` elsewhere for other such "for*Types" to ease the development pain, so you can add whatever you feel like. Could we update this type similarly?

```suggestion
	forMimeTypes: string | Array<string>;
```

You will most likely need to update your code throughout the components to handle it:

```js
const types = Array.isArray(manifest.forMimeTypes) ? manifest.forMimeTypes : [manifest.forMimeTypes];
const filter = types.includes(previewType);
```